pkg/router/dsl_rule.go

```diff
@@ -0,0 +1,81 @@
+package router
+
+import (
+	"context"
+	"time"
+
+	"mosn.io/api"
+	"mosn.io/mosn/pkg/cel"
+	"mosn.io/mosn/pkg/cel/attribute"
+	"mosn.io/mosn/pkg/cel/extract"
+	v2 "mosn.io/mosn/pkg/config/v2"
+	"mosn.io/mosn/pkg/log"
+)
+
+var dslRouterCompiler = cel.NewExpressionBuilder(extract.Attributemanifest, cel.CompatCEXL)
+
+type DslExpressionRouteRuleImpl struct {
```

Comment:
Need to call here https://github.com/mosn/mosn/blob/master/pkg/router/virtualhost.go#L156.

Reply:
ok, my fault.